### DISCUSSION TITLE
Feature Request: Vertical scroll in taskbar

### DISCUSSION THREAD

**Discussion by JarTheHead** [valinet/ExplorerPatcher#2467] [Ideas]
Currently, if the taskbar is full, you can only scroll left to right to view the extra icons. This is especially noticeable when the taskbar is on the left or right side of the screen. Allowing vertical scrolling will make ease of use of the taskbar better, especially for mice without horizontal scrolling.

**Comment by pyrates999** [comment]
If there is existing code to support it, then yes.  If you can find an existing mod that already does it, then yes.

**Comment by sarasassa** [comment]
Retrobar (https://github.com/dremin/RetroBar/) natively supports this behavior. So maybe it's part of the older windows builds functionality?

**Comment by sarasassa** [comment]
https://github.com/user-attachments/assets/fc637ae4-5aec-4087-af94-ab74e0647c73

**Reply by pyrates999** [reply]
I meant a mod for windhawk.
